# Titanic competition with TensorFlow Decision Forests

This notebook will take you through the steps needed to train a baseline Gradient Boosted Trees Model using TensorFlow Decision Forests and creating a submission on the Titanic competition. 

This notebook shows:

1. How to do some basic pre-processing. For example, the passenger names will be tokenized, and ticket names will be splitted in parts.
1. How to train a Gradient Boosted Trees (GBT) with default parameters
1. How to train a GBT with improved default parameters
1. How to tune the parameters of a GBTs
1. How to train and ensemble many GBTs

# Imports dependencies

In [18]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.9.0


# Load dataset

In [19]:
train_df = pd.read_csv("./inputs/train.csv")
serving_df = pd.read_csv("./inputs/test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Prepare dataset

We will apply the following transformations on the dataset.

1. Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
2. Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [20]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

Let's keep the list of the input features of the model. Notably, we don't want to train our model on the "PassengerId" and "Ticket" features.

In [21]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Convert Pandas dataset to TensorFlow Dataset

In [22]:
def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

# Train model with default parameters

### Train model

First, we are training a GradientBoostedTreesModel model with the default parameters.

In [23]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-05-22 11:50:34.4342 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:50:34.4343 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:50:34.4343 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.804347813129425 Loss:0.8922085165977478


[INFO 24-05-22 11:50:34.6408 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpcj_x2r0x/model/ with prefix 981e27946847481f
[INFO 24-05-22 11:50:34.6479 BST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-05-22 11:50:34.6479 BST kernel.cc:1061] Use fast generic engine


# Train model with improved default parameters

Now you'll use some specific parameters when creating the GBT model

In [24]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-05-22 11:50:34.7708 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:50:34.7708 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:50:34.7708 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.77173912525177 Loss:1.0136744976043701


[INFO 24-05-22 11:50:35.0582 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpl3rnlylc/model/ with prefix c9193daba3de41f1
[INFO 24-05-22 11:50:35.0634 BST decision_forest.cc:734] Model loaded with 54 root(s), 2824 node(s), and 10 input feature(s).
[INFO 24-05-22 11:50:35.0635 BST kernel.cc:1061] Use fast generic engine


Let's look at the model and you can also notice the information about variable importance that the model figured out

In [25]:
model.summary()

Model: "gradient_boosted_trees_model_3"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.431313 ################
    2.           "Age"  0.373986 ############
    3.          "Fare"  0.250774 ####
    4.          "Name"  0.225272 ###
    5.   "Ticket_item"  0.182111 
    6.      "Embarked"  0.181387 
    7. "Ticket_number"  0.180897 
    8.        "Pclass"  0.178279 
    9.         "Parch"  0.175344 
   10.         "SibSp"  0.172167 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 39.00000

# Make predictions

In [26]:
def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="./submissions/tf/submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)
!head ./submissions/tf/submission.csv

Submission exported to ./submissions/tf/submission.csv
PassengerId,Survived
892,0
893,0
894,0
895,0
896,0
897,0
898,0
899,0
900,1


# Training a model with hyperparameter tunning

Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).


In [27]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpix1b919x as temporary training directory


[WARNING 24-05-22 11:50:35.4031 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:50:35.4031 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:50:35.4031 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 24-05-22 11:52:41.5152 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpix1b919x/model/ with prefix 382c98e737b14fbf
[INFO 24-05-22 11:52:41.5336 BST decision_forest.cc:734] Model loaded with 26 root(s), 3162 node(s), and 12 input feature(s).
[INFO 24-05-22 11:52:41.5336 BST abstract_model.cc:1344] Engine "GradientBoostedTreesGeneric" built
[INFO 24-05-22 11:52:41.5336 BST kernel.cc:1061] Use fast generic engine


Accuracy: 0.8904109597206116 Loss:0.6763895750045776


In the last line in the cell above, you can see the accuracy is higher than previously with default parameters and parameters set by hand.

This is the main idea behing hyperparameter tuning.

For more information you can follow this tutorial: [Automated hyper-parameter tuning](https://www.tensorflow.org/decision_forests/tutorials/automatic_tuning_colab)

# Making an ensemble

Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the `honest` parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [28]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

[WARNING 24-05-22 11:52:41.6547 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:41.6548 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:41.6548 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:0


[INFO 24-05-22 11:52:41.8537 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpgtc0azjl/model/ with prefix 387644d017e84947
[INFO 24-05-22 11:52:41.8620 BST kernel.cc:1061] Use fast generic engine


[WARNING 24-05-22 11:52:42.0132 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:42.0132 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:42.0133 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:1


[INFO 24-05-22 11:52:42.2347 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpzawtauxi/model/ with prefix 875f58c5f60c4830
[INFO 24-05-22 11:52:42.2451 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:42.3912 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:42.3912 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:42.3913 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:2


[INFO 24-05-22 11:52:42.6156 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp9p084te5/model/ with prefix a33b5adc565542ec
[INFO 24-05-22 11:52:42.6265 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:42.7728 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:42.7728 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:42.7728 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:3


[INFO 24-05-22 11:52:43.0037 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp3oe9phh1/model/ with prefix b1fd734e381d4101
[INFO 24-05-22 11:52:43.0113 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:43.1676 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:43.1677 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:43.1677 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:4


[INFO 24-05-22 11:52:43.3405 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp78u3h6_t/model/ with prefix 75869b7d2a794e74
[INFO 24-05-22 11:52:43.3449 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:43.4993 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:43.4993 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:43.4993 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:5


[INFO 24-05-22 11:52:43.6879 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmph5jf6hx9/model/ with prefix d3f37a72074947ca
[INFO 24-05-22 11:52:43.7003 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:43.8478 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:43.8478 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:43.8478 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:6


[INFO 24-05-22 11:52:44.0509 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpminx41qy/model/ with prefix d6697e63accb4c3d
[INFO 24-05-22 11:52:44.0606 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:44.2156 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:44.2156 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:44.2156 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:7


[INFO 24-05-22 11:52:44.4153 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpfbxas6ln/model/ with prefix f24a3c73c1934112
[INFO 24-05-22 11:52:44.4262 BST kernel.cc:1061] Use fast generic engine


i:8


[WARNING 24-05-22 11:52:44.6232 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:44.6232 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:44.6232 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-05-22 11:52:44.8484 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpcpo8oyzi/model/ with prefix 42949a8bae46419e
[INFO 24-05-22 11:52:44.8603 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:45.0126 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:45.0126 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:45.0126 BST gradient_boosted_trees.cc:1865] "selective_gradient_boostin

i:9


[INFO 24-05-22 11:52:45.2783 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp3mh1vnbm/model/ with prefix e32f4c5c7d4c4abc
[INFO 24-05-22 11:52:45.2848 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:45.4369 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:45.4369 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:45.4369 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:10


[INFO 24-05-22 11:52:45.6344 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmppapqc6jt/model/ with prefix 1d0696f7c8c24fdd
[INFO 24-05-22 11:52:45.6480 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:45.7949 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:45.7949 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:45.7949 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:11


[INFO 24-05-22 11:52:46.0520 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpwm16oezf/model/ with prefix 9afcf812d46640c2
[INFO 24-05-22 11:52:46.0637 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:46.2108 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:46.2108 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:46.2108 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:12


[INFO 24-05-22 11:52:46.4354 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpizv1a677/model/ with prefix bacf124abc05480f
[INFO 24-05-22 11:52:46.4457 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:46.5913 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:46.5913 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:46.5913 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:13


[INFO 24-05-22 11:52:46.9741 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpv3s1ogxj/model/ with prefix 57ad87a4dac14ab8
[INFO 24-05-22 11:52:46.9983 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:47.1659 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:47.1659 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:47.1659 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:14


[INFO 24-05-22 11:52:47.3782 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp1py9u2vq/model/ with prefix 6c3eca6b36c54571
[INFO 24-05-22 11:52:47.3836 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:47.5294 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:47.5295 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:47.5295 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:15


[INFO 24-05-22 11:52:47.7023 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmprfsmcqoy/model/ with prefix ac16a456e0a84869
[INFO 24-05-22 11:52:47.7164 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:47.8632 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:47.8633 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:47.8633 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:16


[INFO 24-05-22 11:52:48.0957 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp82g4298x/model/ with prefix b145ee5518ca4b9b
[INFO 24-05-22 11:52:48.1056 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:48.2636 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:48.2636 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:48.2636 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:17


[INFO 24-05-22 11:52:48.4505 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp3qx06gmo/model/ with prefix 8b0d4dfc59014e03
[INFO 24-05-22 11:52:48.4622 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:48.6194 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:48.6194 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:48.6194 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:18


[INFO 24-05-22 11:52:48.8570 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpdcw1_smz/model/ with prefix 9d799005875047af
[INFO 24-05-22 11:52:48.8696 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:49.0173 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:49.0174 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:49.0174 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:19


[INFO 24-05-22 11:52:49.2723 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpa9lgo8ou/model/ with prefix 8f5dde4adc534d00
[INFO 24-05-22 11:52:49.2781 BST kernel.cc:1061] Use fast generic engine


i:20


[WARNING 24-05-22 11:52:49.6208 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:49.6209 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:49.6209 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-05-22 11:52:49.8135 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpnt6kiijf/model/ with prefix 91e6038f1ddb424f
[INFO 24-05-22 11:52:49.8261 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:49.9778 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:49.9778 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:49.9778 BST gradient_boosted_trees.cc:1865] "selective_gradient_boostin

i:21


[INFO 24-05-22 11:52:50.1824 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpr4t57mpo/model/ with prefix bf63e8d4865041ab
[INFO 24-05-22 11:52:50.1914 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:50.3431 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:50.3431 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:50.3431 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:22


[INFO 24-05-22 11:52:50.5645 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmppi66dg5t/model/ with prefix bb473845db8b46e8
[INFO 24-05-22 11:52:50.5797 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:50.7310 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:50.7310 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:50.7310 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:23


[INFO 24-05-22 11:52:50.9271 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpoyf8d637/model/ with prefix 96ef50bd7a8d447e
[INFO 24-05-22 11:52:50.9391 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:51.0899 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:51.0899 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:51.0900 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:24


[INFO 24-05-22 11:52:51.2766 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpjqgyeeol/model/ with prefix 2cec0e4bc3e74fa1
[INFO 24-05-22 11:52:51.2809 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:51.4354 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:51.4354 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:51.4355 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:25


[INFO 24-05-22 11:52:51.6427 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpysr2airf/model/ with prefix f20385c8bae3463e
[INFO 24-05-22 11:52:51.6572 BST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-05-22 11:52:51.6572 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:51.8066 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:51.8066 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:51.8066 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:26


[INFO 24-05-22 11:52:51.9990 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpvk_bu76x/model/ with prefix c6395528014d4d41
[INFO 24-05-22 11:52:52.0133 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:52.1626 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:52.1626 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:52.1626 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:27


[INFO 24-05-22 11:52:52.4142 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp_a0w1euo/model/ with prefix 91cc83443a7d4883
[INFO 24-05-22 11:52:52.4249 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:52.5732 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:52.5732 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:52.5732 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:28


[INFO 24-05-22 11:52:52.7536 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmphn6rsof_/model/ with prefix cf5d9dbf518842be
[INFO 24-05-22 11:52:52.7635 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:52.9114 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:52.9114 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:52.9114 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:29


[INFO 24-05-22 11:52:53.1038 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpqtftq3sv/model/ with prefix 6a52e8e460694c49
[INFO 24-05-22 11:52:53.1139 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:53.2722 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:53.2723 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:53.2723 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:30


[INFO 24-05-22 11:52:53.4980 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpfjhq6zt7/model/ with prefix 780e107e7f8b4e13
[INFO 24-05-22 11:52:53.5036 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:53.6517 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:53.6518 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:53.6518 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:31


[INFO 24-05-22 11:52:53.9192 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpqcnw2bcg/model/ with prefix afc18215e49a4bf8
[INFO 24-05-22 11:52:53.9312 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:54.0774 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:54.0774 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:54.0774 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:32


[INFO 24-05-22 11:52:54.2745 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpw5xlyndw/model/ with prefix ebb30e7a3a314874
[INFO 24-05-22 11:52:54.2848 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:54.4418 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:54.4418 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:54.4418 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:33


[INFO 24-05-22 11:52:54.6589 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpojj0p6v_/model/ with prefix 321cc51e8540499d
[INFO 24-05-22 11:52:54.6688 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:54.8149 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:54.8149 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:54.8149 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:34


[INFO 24-05-22 11:52:55.0050 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpf0tfshs_/model/ with prefix 52b7c68c35d84c69
[INFO 24-05-22 11:52:55.0148 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:55.1613 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:55.1614 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:55.1614 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:35


[INFO 24-05-22 11:52:55.3727 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpta8ixwsu/model/ with prefix e19a8e85660b4610
[INFO 24-05-22 11:52:55.3781 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:55.5345 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:55.5345 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:55.5345 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:36


[INFO 24-05-22 11:52:55.7156 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpzcdfl7f8/model/ with prefix e96d6ee5a52c4ec4
[INFO 24-05-22 11:52:55.7244 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:55.8738 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:55.8738 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:55.8738 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:37


[INFO 24-05-22 11:52:56.0718 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpuklr2ay0/model/ with prefix 83a4f8282fba4bfc
[INFO 24-05-22 11:52:56.0825 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:56.2309 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:56.2310 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:56.2310 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:38


[INFO 24-05-22 11:52:56.4490 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp8icz2wmw/model/ with prefix 56288f6576d7436c
[INFO 24-05-22 11:52:56.4621 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:56.6083 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:56.6083 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:56.6083 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:39


[INFO 24-05-22 11:52:56.8442 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpmg7xfsg8/model/ with prefix 4f29496f775b4808
[INFO 24-05-22 11:52:56.8564 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:57.0032 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:57.0032 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:57.0032 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:40


[INFO 24-05-22 11:52:57.2077 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpdufzxreh/model/ with prefix 8e670e2ababb487d
[INFO 24-05-22 11:52:57.2212 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:57.3825 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:57.3825 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:57.3826 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:41


[INFO 24-05-22 11:52:57.6157 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpvdd440k9/model/ with prefix dabfbd7efb504848
[INFO 24-05-22 11:52:57.6216 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:57.7690 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:57.7690 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:57.7690 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:42


[INFO 24-05-22 11:52:57.9675 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpkuz821ky/model/ with prefix a79d667379804e35
[INFO 24-05-22 11:52:57.9767 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:58.1264 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:58.1265 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:58.1265 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:43


[INFO 24-05-22 11:52:58.3341 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpnupwrjuj/model/ with prefix 1ccb06323a974d0c
[INFO 24-05-22 11:52:58.3432 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:58.4973 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:58.4973 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:58.4973 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:44


[INFO 24-05-22 11:52:58.6859 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp1bbb06qg/model/ with prefix 56c476acaf854b43
[INFO 24-05-22 11:52:58.6947 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:58.8440 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:58.8440 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:58.8440 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:45


[INFO 24-05-22 11:52:59.0195 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpmr9c4s36/model/ with prefix 6cbb81f6980b4416
[INFO 24-05-22 11:52:59.0280 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:52:59.1835 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:59.1835 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:59.1835 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:46


[INFO 24-05-22 11:52:59.4315 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpxxb4xox_/model/ with prefix ef56beb1e05042ba
[INFO 24-05-22 11:52:59.4365 BST kernel.cc:1061] Use fast generic engine


i:47


[WARNING 24-05-22 11:52:59.7984 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:59.7985 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:52:59.7985 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-05-22 11:53:00.0075 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp4ohtnj86/model/ with prefix aaf30cab63f241fb
[INFO 24-05-22 11:53:00.0197 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:00.1766 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:00.1766 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:00.1766 BST gradient_boosted_trees.cc:1865] "selective_gradient_boostin

i:48


[INFO 24-05-22 11:53:00.3697 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp68r11x8p/model/ with prefix de694fc792c64cf5
[INFO 24-05-22 11:53:00.3795 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:00.5296 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:00.5297 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:00.5297 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:49


[INFO 24-05-22 11:53:00.7388 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpu5lvq5_i/model/ with prefix 99bf5f39f64341c1
[INFO 24-05-22 11:53:00.7511 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:00.9034 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:00.9034 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:00.9034 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:50


[INFO 24-05-22 11:53:01.1619 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmppzc0omq9/model/ with prefix 0d3f7b18e2f440df
[INFO 24-05-22 11:53:01.1748 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:01.3303 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:01.3303 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:01.3303 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:51


[INFO 24-05-22 11:53:01.5974 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp402p1yhy/model/ with prefix f42fc57207ac4da2
[INFO 24-05-22 11:53:01.6047 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:01.7548 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:01.7548 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:01.7548 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:52


[INFO 24-05-22 11:53:01.9490 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmphxkwhqe8/model/ with prefix 22612a1cd23b466e
[INFO 24-05-22 11:53:01.9583 BST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-05-22 11:53:01.9583 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:02.1085 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:02.1085 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:02.1085 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:53


[INFO 24-05-22 11:53:02.3028 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpzta6f5wa/model/ with prefix bfc20e26c7e0491f
[INFO 24-05-22 11:53:02.3119 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:02.4658 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:02.4658 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:02.4658 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:54


[INFO 24-05-22 11:53:02.6788 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpd3jay7xx/model/ with prefix 8fef5f93c8f1433d
[INFO 24-05-22 11:53:02.6930 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:02.8415 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:02.8415 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:02.8415 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:55


[INFO 24-05-22 11:53:03.0746 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpq72rrown/model/ with prefix cc6eee110ea84c4f
[INFO 24-05-22 11:53:03.0918 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:03.2415 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:03.2415 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:03.2415 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:56


[INFO 24-05-22 11:53:03.4495 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpidtk9ia5/model/ with prefix d8f1a97c583346bd
[INFO 24-05-22 11:53:03.4559 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:03.6152 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:03.6152 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:03.6152 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:57


[INFO 24-05-22 11:53:03.7895 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmps3vkakdz/model/ with prefix 6241c3432d9c4297
[INFO 24-05-22 11:53:03.7940 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:03.9428 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:03.9428 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:03.9428 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:58


[INFO 24-05-22 11:53:04.2286 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpogbrxny5/model/ with prefix f01105727fee49b5
[INFO 24-05-22 11:53:04.2596 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:04.4120 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:04.4120 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:04.4120 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:59


[INFO 24-05-22 11:53:04.6831 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpuhwhzw0o/model/ with prefix 967b23f919a64abf
[INFO 24-05-22 11:53:04.6960 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:04.8464 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:04.8464 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:04.8464 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:60


[INFO 24-05-22 11:53:05.0307 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp6trorfxy/model/ with prefix e3380774e82343e3
[INFO 24-05-22 11:53:05.0433 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:05.1923 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:05.1924 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:05.1924 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:61


[INFO 24-05-22 11:53:05.4029 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpdza0e1na/model/ with prefix 7f66ea75cb414082
[INFO 24-05-22 11:53:05.4222 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:05.5799 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:05.5799 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:05.5799 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:62


[INFO 24-05-22 11:53:05.7905 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpteo4n1p6/model/ with prefix a88835d705884e4c
[INFO 24-05-22 11:53:05.7957 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:05.9447 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:05.9448 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:05.9448 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:63


[INFO 24-05-22 11:53:06.1365 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp2f129cp_/model/ with prefix f8371378f1e64d73
[INFO 24-05-22 11:53:06.1470 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:06.2947 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:06.2947 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:06.2947 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:64


[INFO 24-05-22 11:53:06.5103 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp8ji6fmjw/model/ with prefix 41f912dbcc9c4df2
[INFO 24-05-22 11:53:06.5216 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:06.6691 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:06.6691 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:06.6691 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:65


[INFO 24-05-22 11:53:06.8999 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpjdet5jf5/model/ with prefix f7d304f40bc04352
[INFO 24-05-22 11:53:06.9106 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:07.0567 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:07.0567 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:07.0567 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:66


[INFO 24-05-22 11:53:07.2521 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp3m73cbgj/model/ with prefix 34903fd0a20e4e64
[INFO 24-05-22 11:53:07.2641 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:07.4174 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:07.4174 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:07.4175 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:67


[INFO 24-05-22 11:53:07.6491 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpbofal86k/model/ with prefix 659ebe5b22d342e2
[INFO 24-05-22 11:53:07.6543 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:07.8046 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:07.8046 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:07.8047 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:68


[INFO 24-05-22 11:53:08.0397 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp_2bl8x9v/model/ with prefix 667b1485c62946e0
[INFO 24-05-22 11:53:08.0503 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:08.1987 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:08.1987 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:08.1987 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:69


[INFO 24-05-22 11:53:08.4346 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp1_6awh2w/model/ with prefix e3aa4255d6954fbe
[INFO 24-05-22 11:53:08.4452 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:08.5997 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:08.5997 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:08.5997 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:70


[INFO 24-05-22 11:53:08.7897 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpskf0dsl8/model/ with prefix 8911c5bfdece4a00
[INFO 24-05-22 11:53:08.8020 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:08.9504 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:08.9504 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:08.9504 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:71


[INFO 24-05-22 11:53:09.1297 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpnpypkg5g/model/ with prefix 05ea49a4ebf2479d
[INFO 24-05-22 11:53:09.1415 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:09.2933 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:09.2933 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:09.2934 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:72


[INFO 24-05-22 11:53:09.5124 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpyz36w3ja/model/ with prefix 3a7bf664162348c8
[INFO 24-05-22 11:53:09.5212 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:09.6761 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:09.6761 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:09.6762 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:73


[INFO 24-05-22 11:53:09.8450 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmppjab6_7q/model/ with prefix e2758537f9334cc3
[INFO 24-05-22 11:53:09.8488 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:09.9961 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:09.9961 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:09.9961 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:74


[INFO 24-05-22 11:53:10.2842 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmptutm8sx6/model/ with prefix 08a6cfb5cb574944
[INFO 24-05-22 11:53:10.3020 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:10.4497 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:10.4498 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:10.4498 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:75


[INFO 24-05-22 11:53:10.6329 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpkfu275z2/model/ with prefix 29b48d5f38c448cd
[INFO 24-05-22 11:53:10.6413 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:10.7885 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:10.7885 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:10.7885 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:76


[INFO 24-05-22 11:53:10.9710 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmps_xt98_u/model/ with prefix fcf79695c08845dc
[INFO 24-05-22 11:53:10.9795 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:11.1265 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:11.1265 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:11.1266 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:77


[INFO 24-05-22 11:53:11.5875 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp0pugb30m/model/ with prefix 284bc03e06834b7a
[INFO 24-05-22 11:53:11.5945 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:11.7568 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:11.7568 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:11.7568 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:78


[INFO 24-05-22 11:53:11.9348 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp8pienjvt/model/ with prefix cd1f7f9bdf6049e2
[INFO 24-05-22 11:53:11.9396 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:12.1038 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:12.1038 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:12.1038 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:79


[INFO 24-05-22 11:53:12.3392 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpeat74srr/model/ with prefix 1bfabb549d1941d6
[INFO 24-05-22 11:53:12.3521 BST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-05-22 11:53:12.3521 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:12.5066 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:12.5066 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:12.5066 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:80


[INFO 24-05-22 11:53:12.7039 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpf_oxttq0/model/ with prefix 4744067038934a19
[INFO 24-05-22 11:53:12.7179 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:12.8721 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:12.8721 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:12.8721 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:81


[INFO 24-05-22 11:53:13.0875 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp97uxarju/model/ with prefix 3ad857b981e04542
[INFO 24-05-22 11:53:13.1009 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:13.2519 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:13.2519 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:13.2520 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:82


[INFO 24-05-22 11:53:13.4823 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp64r4ud_1/model/ with prefix 54dbb39cb9184e84
[INFO 24-05-22 11:53:13.4954 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:13.6648 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:13.6649 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:13.6649 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:83


[INFO 24-05-22 11:53:13.9057 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpe5l_85g4/model/ with prefix cffe5b3bfd664312
[INFO 24-05-22 11:53:13.9113 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:14.0623 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:14.0623 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:14.0623 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:84


[INFO 24-05-22 11:53:14.2887 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpbergf10q/model/ with prefix a34a9d8233fc4ec6
[INFO 24-05-22 11:53:14.2938 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:14.4439 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:14.4439 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:14.4439 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:85


[INFO 24-05-22 11:53:14.6692 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpogw8yrg6/model/ with prefix 2575d4fbcff04e02
[INFO 24-05-22 11:53:14.6836 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:14.8326 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:14.8326 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:14.8326 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:86


[INFO 24-05-22 11:53:15.1066 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpmtnth500/model/ with prefix f00752caa30c4f3a
[INFO 24-05-22 11:53:15.1231 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:15.2826 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:15.2826 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:15.2827 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:87


[INFO 24-05-22 11:53:15.4942 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp0ajoqojj/model/ with prefix ae68641894b34572
[INFO 24-05-22 11:53:15.5058 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:15.6706 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:15.6706 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:15.6706 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:88


[INFO 24-05-22 11:53:15.8447 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmps27d0rbp/model/ with prefix 1627e406af784a12
[INFO 24-05-22 11:53:15.8483 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:16.0000 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:16.0001 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:16.0001 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:89


[INFO 24-05-22 11:53:16.2093 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmplj5lxlq8/model/ with prefix 572d04b83c944a4a
[INFO 24-05-22 11:53:16.2213 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:16.3695 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:16.3696 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:16.3696 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:90


[INFO 24-05-22 11:53:16.5695 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpdj47x_ea/model/ with prefix 8ba6737eadb64539
[INFO 24-05-22 11:53:16.5822 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:16.7335 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:16.7335 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:16.7335 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:91


[INFO 24-05-22 11:53:16.9703 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpd7z42j0f/model/ with prefix 2c623c348efb47d5
[INFO 24-05-22 11:53:16.9879 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:17.1384 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:17.1384 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:17.1384 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:92


[INFO 24-05-22 11:53:17.4656 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp6s5wd0ow/model/ with prefix cb6c5f1e423a4684
[INFO 24-05-22 11:53:17.4867 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:17.6381 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:17.6382 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:17.6382 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:93


[INFO 24-05-22 11:53:17.8342 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp73r9c9kk/model/ with prefix 60c84627a2d64220
[INFO 24-05-22 11:53:17.8384 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:17.9974 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:17.9974 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:17.9975 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:94


[INFO 24-05-22 11:53:18.2042 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpvnxlcgf2/model/ with prefix 2585585211e74151
[INFO 24-05-22 11:53:18.2188 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:18.3679 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:18.3679 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:18.3679 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:95


[INFO 24-05-22 11:53:18.5714 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp1bf8ra3x/model/ with prefix 8a03b39281874400
[INFO 24-05-22 11:53:18.5809 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:18.7299 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:18.7300 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:18.7300 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:96


[INFO 24-05-22 11:53:18.9394 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpm01dbg8y/model/ with prefix d3d4cfb51cb84ef0
[INFO 24-05-22 11:53:18.9490 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:19.0959 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:19.0960 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:19.0960 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:97


[INFO 24-05-22 11:53:19.2828 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp13nvosyt/model/ with prefix 0f77eb9c8aa04849
[INFO 24-05-22 11:53:19.2932 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:19.4410 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:19.4410 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:19.4410 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:98


[INFO 24-05-22 11:53:19.7894 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmpdzf729w5/model/ with prefix 52f4653b31a94c92
[INFO 24-05-22 11:53:19.7989 BST kernel.cc:1061] Use fast generic engine
[WARNING 24-05-22 11:53:19.9528 BST gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:19.9528 BST gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-05-22 11:53:19.9528 BST gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:99
Submission exported to ./submissions/tf/submission.csv


[INFO 24-05-22 11:53:20.3317 BST kernel.cc:1233] Loading model from path /var/folders/km/2_7xtqxn6x94c9qsgqjs2jc0_h98zq/T/tmp1ajrp6zj/model/ with prefix b1f5c419b21940d6
[INFO 24-05-22 11:53:20.3532 BST kernel.cc:1061] Use fast generic engine


# What is next

If you want to learn more about TensorFlow Decision Forests and its advanced features, you can follow the official documentation [here](https://www.tensorflow.org/decision_forests) 